projects/Allwinner/filesystem/usr/share/alsa/cards/simple-card.conf

```diff
@@ -71,6 +71,8 @@ simple-card.pcm.hdmi.0 {
 
 <confdir:pcm/iec958.conf>
 
+simple-card.pcm.iec958."On-board SPDIF" simple-card.pcm.iec958."sun50i-h6-spdif"
```

Comment:
You to prefix the referred card config with the "cards." namespace and the examples have shown it needs to be completly quoted, i.e. "cards.simple-card.pcm.iec958.sun50i-h6-spdif"

Reply:
Fixed.

Reply:
scratch that

Reply:
now it's fixed